api/src/main/java/uk/gov/dvsa/mot/motr/model/VehicleType.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.dvsa.mot.motr.model;
+
+public enum VehicleType {
+    MOT,
+    HGV,
+    PSV;
```

Comment:
potentaily we could use these types in HGV response + Trailer type

Reply:
potentialy for later enhancement